src/content/app/genome-browser/hooks/useGenomeBrowser.ts

```diff
@@ -286,7 +305,7 @@ const useGenomeBrowser = () => {
     });
   };
 
-  const toggleTrackLabel = (params: {
+  const toggleFeatureLabel = (params: {
```

Comment:
- why `feature` rather than specifically `transcript`? the message we are sending to the genome browser inside this function is specific to transcripts
- why `label` (singular) rather than `labels` (plural), or `ids`?

Reply:
this is a generic method to turn on label, not just for transcript. It can be variant, GC, contig etc.

Reply:
label still not renamed to labels